### DISCUSSION TITLE
mount: /tmp/wsa-build-UaHRP9AcMI_/system: mount failed: Operation not permitted.

### DISCUSSION THREAD

**Discussion by chenxinxun** [LSPosed/MagiskOnWSALocal#377] [Q&A]
1.run run.sh
2. then result show: mount: /tmp/wsa-build-UaHRP9AcMI_/system: mount failed: Operation not permitted.

**Comment by Howard20181** [comment]
What your build environments?

**Reply by jkwangyuxi** [reply]
Mount images
mount: /tmp/wsa-build-DGCKH3eqCQ_/system: mount failed: Operation not permitted.
Build: an error has occurred, exit
